### PR TITLE
Fix the 'When no star ratings on book, UI looks weird #6411' bug

### LINKED ISSUE #6411: When no star ratings on book, UI looks weird
### Evidence / Screenshot (if possible)
image

### Relevant url?
eg https://openlibrary.org/books/OL664495M/An_octave_above_thunder

### Steps to Reproduce

1. Go to book with no ratings

### Details

- **Logged in (Y/N)?**
- **Browser type/version?**
- **Operating system?**
- **Environment (prod/dev/local)?** prod

### PR DESCRIPTION
**Pull request by Vassilis-Boubis**
Closes #6411 

### Screenshot
Screenshot 2022-04-12 at 11 43 05 PM
Screenshot 2022-04-12 at 11 42 43 PM

### CODE REVIEW

File: openlibrary/macros/StarRatingsStats.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
       $if (stats_decimal >= 0.5) and (stats_decimal < 1):
         $:('<span class="readers-stats__star--half">★</span>')
       <span itemprop="ratingValue">$avg</span>
+      <h4>·</h4>
```
Comment (jimchamp) [inline openlibrary/macros/StarRatingsStats.html:15]:
```suggestion
      <span>·</span>
```
Let's change this to a span, and update the CSS accordingly. You can assign the span a unique class if it is helpful.

**Review by jimchamp — CHANGES_REQUESTED** [review]
Headings (`<h1>,...,<h6>`) are meant to create an outline of the content on a page.  Pages are less accessible when headings are used as decorative elements.

File: openlibrary/macros/StarRatingsStats.html (hunk 1 above)
Comment (Vassilis-Boubis) [reply openlibrary/macros/StarRatingsStats.html:15]: Okey, I will fix it rigth now

**Review by jimchamp — COMMENTED** [review]
Lgtm!  Thanks @Vassilis-Boubis!

**Pull request merged**